keymaps/ascii-art.json

```diff
@@ -0,0 +1,5 @@
+{
+  "atom-text-editor": {
+    "ctrl-alt-a": "ascii-art:convert"
+  }
+}
```

Comment:
Are you sure we want to convert the keymap from CSON to JSON? I haven't seen that done in other packages. Even new [packages like atom/github still use CSON for their keymaps](https://github.com/atom/github/tree/1eff46c4046af167b0e0686a18b3c9ac0898d486/keymaps).

Reply:
The reason why I would want to convert the keymaps file from `.cson` -> `.json` is because that's what the package generator generates by default.  

![image](https://user-images.githubusercontent.com/5624870/31101963-e1d7de2a-a784-11e7-912a-4385c6c4a780.png)

Reply:
@ungb: That's I darn good reason. 😸  Thanks for letting me know. :bow: